sources/install.sh

```diff
@@ -1337,15 +1337,15 @@ function install_pwnedornot() {
   add-test-command "pwnedornot.py --help"
 }
 
+function install_chromium() {
+  apt-get install -y chromium
```

Comment:
You need to use `fapt` instead of `apt-get install -y` -> `fapt chromium`

Reply:
Yes thank you, done !